src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisShardSyncer.java

```diff
@@ -788,9 +629,9 @@ static KinesisClientLease newKCLLease(Shard shard) {
      * @return
      */
     static KinesisClientLease newKCLLeaseForChildShard(ChildShard childShard) throws InvalidStateException {
-        final KinesisClientLease newLease = new KinesisClientLease();
+        KinesisClientLease newLease = new KinesisClientLease();
         newLease.setLeaseKey(childShard.getShardId());
-        final List<String> parentShardIds = new ArrayList<>();
+        List<String> parentShardIds = new ArrayList<>();
```

Comment:
why finals were removed?

Reply:
... not sure, will unstage this change...